src/types/traceback.rs

```diff
@@ -119,10 +169,13 @@ def f():
             .unwrap();
             let f = locals.get_item("f").unwrap().unwrap();
             let err = f.call0().unwrap_err();
-            let traceback = err.traceback(py).unwrap();
+            let traceback = err.traceback_bound(py).unwrap();
             let err_object = err.clone_ref(py).into_py(py).into_ref(py);
 
-            assert!(err_object.getattr("__traceback__").unwrap().is(traceback));
+            assert!(err_object
+                .getattr("__traceback__")
+                .unwrap()
+                .is(traceback.as_gil_ref()));
```

Comment:
Does it work if we remove the `.as_gil_ref()`, and insert `&` instead? It'd be nice to avoid going back to the old API here. If it doesn't compile, it would be great for us to think how it can work (maybe as a future PR).

```suggestion
                .is(&traceback));
```

Reply:
This works! Even better.